### ISSUE TITLE
React Native Dev tool lauching error

### ISSUE THREAD

**Issue by varunajmera0** [react-community/create-react-native-app#558]
I am requesting to launch dev tool but it forward to http://10.0.2.2:8081/debugger-ui

![image](https://user-images.githubusercontent.com/22893376/35977215-7333c6a2-0d08-11e8-9e15-2fa6d56b8f29.png)

![image](https://user-images.githubusercontent.com/22893376/35977177-59624d66-0d08-11e8-92c5-70f0f8b6a565.png)

![image](https://user-images.githubusercontent.com/22893376/35977064-18719c94-0d08-11e8-8f9e-8ff605caad12.png)

**Comment by Alastair-smith2** [comment]
Haven't run into this myself, just happen to see the same issue raised over at the react-native repo, here's the link - https://github.com/facebook/react-native/issues/17970, hope it helps

**Comment by devmrin** [comment]
Solved the issue for me following:

Press Cmd + M on emulator screen
Go to Dev settings > Debug server host & port for device
Set localhost:8081
Rerun the android app: react-native run-android
Debugger is connected now!

**Comment by manualva7** [comment]
Changing the debug server host & port for device to locahost:8081 doesn't work either. It does work if I enter the address in my browser.

(EDIT)

After looking closely at terminal I saw ADB wasn't being recognized. I added the address of the adb file inside platform-tools into the PATH environment variable, changed the dev inside the emulator to localhost:8081 and it worked. Not sure how it worked without that environment path, but now I'm sure that without it, remote debug doesn't work no matter what you change.
